tests/echo_files_parser_test.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+from unittest.mock import patch, mock_open, MagicMock
+from src.data.file_preprocessing.echo_files_parser import EchoFilesParser
+import pandas as pd
+
+
+class TestEchoFilesParser(unittest.TestCase):
```

Comment:
not sure if we'd want to use two testing frameworks (`pytest` and `unittest`) for our unit tests - especially considering we use pytest in CI/CD pipeline so I'm not sure if they will be picked up correctly

Reply:
Np, will change that